### DISCUSSION TITLE
Let us configure the badge labels

### DISCUSSION THREAD

**Discussion by antoine-pous** [community/community#25425] [Actions]
<p>Hi!</p>
<p>It would be really awesome to let us configure the badge passing/fail labels. I’ve actually one workflow to publish my package on NPM and another one to GPR.</p>
<p>It would be awesome to being able to setting up what we want display</p>
<pre><code class="lang-auto">badge:
  label: NPM release # Displayed label, instead of picking our internal CI name
  passing: ${{ package.version }} # Displayed value, can be related to the package.json etc...
  failing: 
  ignore-ci-failure: false # if true, then the badge is not updated and will refer to the last successful job if the current job fail
</code></pre>
<p>IMO badges should offer this kind of feature, they’ll become incredibly useful with that :slight_smile:</p>
<p>Thanks for your work :+1:</p>

**Comment by BrightRan** [comment, marked as answer]
<p><a class="mention" href="/antoine-pous">@antoine-pous</a>,</p>
<p>Thanks for your feedback.</p>
<p>GitHub takes your suggestions very seriously, and the suggestions are very helpful for improving GitHub.</p>
<p>I recommend that you can directly share your feature requests <a href="https://support.github.com/contact/feedback?contact%5Bcategory%5D=actions" rel="noopener nofollow ugc">here</a>. That will allow you to directly interact with the appropriate engineering team, and make it more convenient for the engineering team to collect and categorize your suggestions.</p>

**Comment by antoine-pous** [comment]
<p>I’ve done it, can i expect a status update about it from the team?</p>

**Comment by BrightRan** [comment]
<p><a class="mention" href="/antoine-pous">@antoine-pous</a>,</p>
<p>The engineering team will send their replies to your email box, and you also can directly reply them on the email message.</p>

**Comment by fkirc** [comment]
<p>Customized Badge-Labels would be a great feature. Right now, you need to either configure and maintain an external service, or rename the Workflow within a YML-file.</p>
